README.md

```diff
@@ -5,24 +5,31 @@ Create, fork and edit d3.js code snippets for use with bl.ocks.org right in the
 Eventually we will host this with a domain, but for now let's just get it running locally by following the instructions in Development.
 
 # Development
-
-MongoDB and Redis are required (for sessions and users), they are currently configured inside `server.js`
+MongoDB and Redis are required (for sessions and users) so make sure you install them before proceeding any further. If you would like to tweak the default parameters, you can do so my modifying `server.js`.
 
 ```bash
 git clone https://github.com/enjalot/building-blocks.git
 cd building-blocks/
 npm install
-# run webpack to build the js & css bundles
+# run webpack to build the js & css bundles (note that this does not terminate, so feel free to kill this when no apparent progress is being made)
 npm run local
-# run the server (use node-dev server.js for auto-restarting on changes to server code)
-node server.js
+# now set up mongodb with the following commands
+mkdir -p /data/db
```

Comment:
this shouldn't be necessary, or rather, we shouldn't specify one of the many ways to run mongo. I think it would be better to link to a tutorial on running a mongo server in various environments.